com.jetpackduba.Gitnuro.yml

```diff
@@ -0,0 +1,38 @@
+app-id: com.jetpackduba.Gitnuro
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: gitnuro
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk17/install.sh
+  - name: gitnuro
+    buildsystem: simple
+    build-commands:
+      - install -D gitnuro /app/bin/gitnuro
+      - install -D Gitnuro-0.1.0.jar /app/share/Gitnuro.jar
+      - install -D com.jetpackduba.Gitnuro.desktop /app/share/applications/com.jetpackduba.Gitnuro.desktop
+      - install -D icon.png /app/share/icons/hicolor/512x512/apps/com.jetpackduba.Gitnuro.png
+      - install -D com.jetpackduba.Gitnuro.metainfo.xml /app/share/appdata/com.jetpackduba.Gitnuro.metainfo.xml
+    sources:
+      - type: file
+        path: gitnuro
+      - type: file
+        url: https://github.com/JetpackDuba/Gitnuro/releases/download/v0.1.0/Gitnuro-0.1.0.jar
+        sha256: 9502032cf103b9a31bdaf1a8f0495d23917ecac38b43a2c34bb882f070b47a63
+      - type: file
+        path: com.jetpackduba.Gitnuro.desktop
+      - type: file
+        path: icon.png
+      - type: file
+        path: com.jetpackduba.Gitnuro.metainfo.xml
+finish-args:
```

Comment:
this is usually above "modules" as a common practice

Reply:
Do you mean the `finish-args`?